docs/advanced_topics/index.rst

```diff
@@ -8,7 +8,7 @@ Advanced topics
     images/index
     documents/index
     embeds
-    settings
+    addtodjangoproject
```

Comment:
Let's use underscore-separated words here, for consistency with other pages (e.g. `third_party_tutorials`).